connector-setup/index.js

```diff
@@ -152,7 +154,8 @@ exports.run = function (workingPath, callback) {
           anonymousSearchEnabled(isEnabled);
         })
         .once('error',function(err){
-          anonymousSearchEnabled(false);
+          // if there are more than one entry matching the search, the server returns the one entry and a SizeLimitExceededError error
+          anonymousSearchEnabled(err.name === 'SizeLimitExceededError');
```

Comment:
Should it be instead `err.name === 'SizeLimitExceededError' && searchEntry`  to avoid edge case where server time limit or size limit exceeds even while fetching 1 record.

Reply:
if server time limit, then err.name will be different

Reply:
got it. So, if time limit error comes, we set anonymousSearchEnabled to false.  Would it be safe to assume anonymousSearchEnabled as true in case of `SizeLimitExceededError ` error but when searchEntry is null? Not sure if it is a valid scenario though.

Reply:
FWIK, if you receive `SizeLimitExceededError` means that the query worked, so we should enable anonymous search.

```
Size Limit Exceeded is an indication that you are getting multiple search results using your search filter for your LDAP Adapter or LDAP Attribute lookup.
```

Reply:
Makes sense. Thank you.